packages/python/plotly/test_requirements/requirements_312_core.txt

```diff
@@ -1,3 +1,4 @@
 requests==2.25.1
 tenacity==6.2.0
 pytest==7.4.4
+numpy==1.23.2
```

Comment:
Why we have to pin down this version here?

Reply:
Oh that totally fixed the CI. Thanks!